### PR TITLE
docs: Edits to remove jargon

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/testing-components-scenarios.md [hunk 1]
```diff
@@ -1135,7 +1135,7 @@ Here's a test demonstrating the component's behavior when the observed `id` refe
 
 <div class="alert is-helpful">
 
-The `createComponent()` method and `page` object are discussed [below](#page-object).
+The `createComponent()` method and `page` object are discussed [following](#page-object).
```
Comment (AndrewKushnir) [inline aio/content/guide/testing-components-scenarios.md:1138]: That looks like an incomplete sentence too:
```suggestion
The `createComponent()` method and `page` object are discussed following.
```

File: aio/content/guide/testing-components-scenarios.md [hunk 2]
```diff
@@ -1174,16 +1174,16 @@ The `AppComponent`, for example, displays a navigation bar with anchors and thei
   header="app/app.component.html"></code-example>
 
 While the `AppComponent` _class_ is empty,
-you may want to write unit tests to confirm that the links are wired properly
-to the `RouterLink` directives, perhaps for the reasons [explained below](#why-stubbed-routerlink-tests).
+you might want to write unit tests to confirm that the links are wired properly
+to the `RouterLink` directives, perhaps for the reasons as [explained](#why-stubbed-routerlink-tests).
```
Comment (AndrewKushnir) [inline aio/content/guide/testing-components-scenarios.md:1178]:
```suggestion
to the `RouterLink` directives, perhaps for the reasons as explained in the [following section](#why-stubbed-routerlink-tests).
```

**Review by aikithoughts — COMMENTED** [review]
Looking good. Some comments from AndrewKushnir need to be addressed. Then we can move forward!

File: aio/content/guide/testing-components-scenarios.md [hunk 3]
```diff
@@ -394,7 +394,7 @@ There is no nested syntax (like a `Promise.then()`) to disrupt the flow of contr
 <div class="alert is-helpful">
 
 Limitation: The `fakeAsync()` function won't work if the test body makes an `XMLHttpRequest` (XHR) call.
-XHR calls within a test are rare, but if you need to call XHR, see [`waitForAsync()`](#waitForAsync), below.
+XHR calls within a test are rare, but if you need to call XHR, see [`waitForAsync()`](#waitForAsync), section.
```
[the remark this answers is not in the document]
Comment (TeriGlover) [reply aio/content/guide/testing-components-scenarios.md:397]: Done

File: aio/content/guide/testing-components-scenarios.md [hunk 4]
```diff
@@ -424,18 +424,18 @@ It's a companion to `fakeAsync()` and you can only call it within a `fakeAsync()
   region="fake-async-test-tick-new-macro-task-sync">
 </code-example>
 
-In this example, we have a new macro task (nested setTimeout), by default, when we `tick`, the setTimeout `outside` and `nested` will both be triggered.
+In this example, you have a new macro task (nested setTimeout), by default, when you `tick`, the setTimeout `outside` and `nested` will both be triggered.
```
[the remark this answers is not in the document]
Comment (TeriGlover) [reply aio/content/guide/testing-components-scenarios.md:427]: Done

File: aio/content/guide/testing-components-scenarios.md (hunk 1 above)
Comment (TeriGlover) [reply aio/content/guide/testing-components-scenarios.md:1138]: In the following section, the `createComponent()` method and `page` object are discussed.

File: aio/content/guide/testing-components-scenarios.md (hunk 2 above)
Comment (TeriGlover) [reply aio/content/guide/testing-components-scenarios.md:1178]: Done

File: aio/content/guide/testing-components-scenarios.md [hunk 5]
```diff
@@ -171,7 +171,7 @@ Please call "TestBed.compileComponents" before your test.
 You get this test failure message when the runtime environment
 compiles the source code _during the tests themselves_.
 
-To correct the problem, call `compileComponents()` as explained [below](#compile-components).
+To correct the problem, call `compileComponents()` as [explained](#compile-components).
```
[the remark this answers is not in the document]
Comment (TeriGlover) [reply aio/content/guide/testing-components-scenarios.md:174]: Done

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**